src/config.c

```diff
@@ -788,19 +799,26 @@ static int core_config_template(struct re_printf *pf, const struct config *cfg)
 	have_cafile = true;
 #endif
 
+#if defined (DEFAULT_CAFILE) || defined (LINUX)
```

Comment:
If `DEFAULT_CAFILE` is set, then for non-Linux do we want
```
sip_cafile    DEFAULT_CAFILE
#  and
sip_capath   /etc/ssl/certs
```

Reply:
Yes, sorry, should have been `DEFAULT_CAPATH` instead of `DEFAULT_CAFILE`, corrected.